router/image/templates/nginx.conf

```diff
@@ -157,6 +158,12 @@ http {
 
             proxy_next_upstream         error timeout http_502 http_503 http_504;
 
+            {{ if ne $enforceHttps "false" }}
```

Comment:
As a followup, just above this section, there is:

    {{ if ne $useSSL "false" }}

This is the reason I had used this method of comparison, the other similar references in this template used this same "trick". Is it idiomatic? Is it necessary? If the bare case works, I'm fine with the clearer expression.